kong/consumer_group_test.go

```diff
@@ -242,3 +242,106 @@ func compareConsumerGroups(expected, actual []*ConsumerGroup) bool {
 
 	return (compareSlices(expectedNames, actualNames))
 }
+
+func TestConsumerGroupGetEndpointPreGW39(t *testing.T) {
+	RunWhenEnterprise(t, ">=2.7.0 <3.9.0", RequiredFeatures{})
+	assert := assert.New(t)
+
+	client, err := NewTestClient(nil, nil)
+	require.NoError(t, err)
+	assert.NotNil(client)
+
+	// create consumer
+	consumer := &Consumer{
+		Username: String("foo"),
+	}
+	createdConsumer, err := client.Consumers.Create(defaultCtx, consumer)
+	require.NoError(t, err)
+	assert.NotNil(createdConsumer)
+	t.Cleanup(func() {
+		err = client.Consumers.Delete(defaultCtx, createdConsumer.ID)
+		assert.NoError(err)
```

Comment:
Might as well do these, WDYT?

```suggestion
		assert.NoError(client.Consumers.Delete(defaultCtx, createdConsumer.ID))
```